specification/redis/resource-manager/Microsoft.Cache/stable/2015-08-01/redis.json

```diff
@@ -451,7 +451,7 @@
         },
         "staticIP": {
           "type": "string",
-          "description": "Required when deploying a Redis cache inside an existing Azure Virtual Network."
+          "description": "Deprecated. May optionally be used to request a specific IP address, only when deploying a Redis cache inside an Azure virtual network. We recommend you do not set this parameter, and connect using the hostname instead."
```

Comment:
hi @TimLovellSmith just to double confirm, is the deprecation happens in the `2015-08-01` service tag or newer version?

Reply:
While the property still usually works, we would prefer to change our guidance to be for people to prefer not to use this property with all api versions.

Reply:
does that mean you'll update description of all versions ?

Reply:
Yes, the description in all versions is being updated as part of this PR.